app/components/Message/index.js

```diff
@@ -207,6 +86,162 @@ function Message({ extended, type, payload, user, msgForm }) {
   );
 }
 
+function InviteFromMe() {
```

Comment:
Shouldn't these new functions take `payload`, `user` and `msgForm` as arguments ?

Reply:
This ^

Reply:
@marzavec @raf924 should be fine now

Reply:
Still not. Please test your code.
If you call `MyFunction(payload)` then you can't define `function MyFunction({payload})`. That means your function expects an object with the property `payload`.

Reply:
@raf924 tested, pushed.

Reply:
@marzavec done

Reply:
Fixed /whisper bug that displayed same message when whispered yourself

Reply:
WIP - will fix formatting